### PR TITLE
Fix: Hide affiliations when agent is organization in agent form

### PR DESCRIPTION
**Pull request by Bilelkihal**
Related issue:
https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/589

### CODE REVIEW

**Review by Bilelkihal — COMMENTED** [review]
✅

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Tests don't pass

**Comment by Bilelkihal** [comment]
> Tests don't pass

Not related to this PR

**Comment by syphax-bouazzouni** [comment]
> > Tests don't pass
> 
> Not related to this PR

This one is related to this PR

image

File: test/system/submission_flows_test.rb
Comment (syphax-bouazzouni) [inline test/system/submission_flows_test.rb:1]: Still, the test don't pass
image

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Review by syphax-bouazzouni — APPROVED** [review]
Code approved and fixed the remaining tests in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/696/commits/667f8b0917bf5cfea09c95f32ffb787ccc9690b9

**Pull request merged**